dist/pathfora.js

```diff
@@ -2910,7 +2910,19 @@
 
       this.getData(recommendUrl, function (json) {
         var resp = JSON.parse(json);
+
         if (resp.data && resp.data.length > 0) {
+          // append a protocol for urls that are absolute
+          for (var i = 0; i < resp.data.length; i++) {
+            var url = resp.data[i].url;
+            if (url) {
+              var split = url.split('/')[0].split('.');
+              if (split.length > 1) {
+                resp.data[i].url = 'http://' + url;
```

Comment:
i could be derping but are we sure this will always be insecure? i realize we can't just pull the current protocol and append that for a recommendation that is not necessarily in the same environment but seems like worst case its a ticket we should make related to content recommendations? kinda seems yuck for us to be manipulating urls to something that is being returned from an api? 

to be clear, fine for now but seems like it smells bad.